modules/proc-configuring-container-registry-authn.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following:
+//
+// @smccarthy-ie - Getting Started with Camel Kafka Connector
+//
+// Attributes used:
+//
+// {ProductName}
+// {OCPVersion}
+
+
+[id="configuring-container-registry-authn_{context}"]
+= Configuring authentication with registry.redhat.io
+
+You must configure authentication with `registry.redhat.io` before you can deploy {ProductName} container images on OpenShift.
+
+.Prerequisites
+* You must have cluster administrator access to an OpenShift Container Platform cluster.  
```

Comment:
Depends I guess on the style you use in your guides, but we avoid ' you' in front of these.

```suggestion
* Have cluster administrator access to an OpenShift Container Platform cluster.  
```